src/v3/src/components/ConsentHeader/ConsentHeader.tsx

```diff
@@ -106,7 +92,7 @@ const ConsentHeader: FunctionComponent = () => {
   const getHeaderContent = () => {
     if ([IDX_STEP.CONSENT_ADMIN, IDX_STEP.CONSENT_ENDUSER].includes(stepName)) {
       // @ts-expect-error OKTA-598777 authentication missing from IdxContext interface
-      const { context: { authentication: { value: { issuer } } = {} } } = idxTransaction;
+      const { rawIdxState: { authentication: { value: { issuer } } = {} } } = idxTransaction;
```

Comment:
was `context` also missing data here?

Reply:
Yes it was, not sure why, but it was missing client URI data.